nixos/default.nix

```diff
@@ -5,14 +5,26 @@ with lib;
 let
 
   cfg = config.home-manager;
-
-  serviceEnvironment = optionalAttrs (cfg.backupFileExtension != null) {
-    HOME_MANAGER_BACKUP_EXT = cfg.backupFileExtension;
-  } // optionalAttrs cfg.verbose { VERBOSE = "1"; };
+  baseService = username: {
+    Type = "oneshot";
+    RemainAfterExit = "yes";
+    TimeoutStartSec = 90;
+    SyslogIdentifier = "hm-activate-${username}";
+  };
+  baseUnit = username: {
+    description = "Home Manager environment for ${username}";
```

Comment:
Is `%u` supported there?

Reply:
Then `baseUnit "%u"` won't work.

Reply:
wait why not?

Reply:
Because that will set `description = "Home Manager environment for %u";`.